hatch/src/constants.rs

```diff
@@ -0,0 +1,22 @@
+pub static ARGS: &str = "ARGS";
```

Comment:
This file is the javascript developer coming out...I like the idea of a single file with constants. I think it is good for a small project like this.